src/slurm_plugin/clustermgtd.py

```diff
@@ -137,6 +137,11 @@ class ClustermgtdConfig:
         "disable_scheduled_event_health_check": False,
         "disable_all_cluster_management": False,
         "health_check_timeout": 180,
+        # DNS domain configs
+        "dynamodb_table": None,
+        "hosted_zone": None,
+        "dns_domain": None,
+        "use_private_hostname": "no",
```

Comment:
let's translate this in the cookbook to true/false

Reply:
Changed, we're now using True/False in the node code, and "true"/"false" in the cookbook one.

I tested it with `"use_private_hostname": "true"`  and `"use_private_hostname": True` and with the default value.